lldb/source/Plugins/Process/Dpu/ProcessDpu.cpp

```diff
@@ -50,6 +50,10 @@
 #include "Plugins/Process/POSIX/ProcessPOSIXLog.h"
 #include "ThreadDpu.h"
 
+extern "C" {
+#include <dpu_program.h>
```

Comment:
readjust header includes.
have together the backends includes, (dpu_types.h above)